src/presence_analyzer/utils.py

```diff
@@ -199,3 +199,65 @@ def star_end_time(data, user_id):
         result[day]['end'] = average_seconds(result[day], 'end')
 
     return result
+
+
+def bussines_days(year_month):
+    """
+    Calculate soconds of worked days in month.
+    """
+    year, month = year_month
+    eight_hour = 28800
```

Comment:
this isn't clear if u don't check if it is 8 hours, no comment etc... 
thats how you should decrare it:
xxx = 60*60*8